addons/udes_stock/tests/test_move_line.py

```diff
@@ -0,0 +1,108 @@
+from . import common
+from odoo.exceptions import ValidationError
+
+
+# NB(ale): the drop location constraint is also tested in
+# test_update_picking::TestUpdatePickingMarksMoveLinesAsDone
+# via Picking::update_picking
+
+class TestValidateLocationDest(common.BaseUDES):
+    ''' Tests the drop location constraint '''
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestValidateLocationDest, cls).setUpClass()
+        cls.picking_type_in.u_target_storage_format = 'product'
+        cls._pick_info = [{'product': cls.apple, 'qty': 4}]
+        cls._constraint = cls.picking_type_putaway.u_drop_location_constraint
+        cls._policy     = cls.picking_type_putaway.u_drop_location_policy
```

Comment:
should only one space around an "="

Reply:
Ah, you don't like vertical alignment, ok, my eyes will blame you.

Reply:
pylint(pep8) doesn't like it. Sorry

Reply:
Ah, like we're pep8ing everywhere :D

Reply:
We should be :laughing:

Reply:
Also why I suggested using an auto-formatter.